spec.html

```diff
@@ -40168,14 +39996,13 @@ <h1>
       <emu-clause id="sec-validateatomicaccess" type="abstract operation">
         <h1>
           ValidateAtomicAccess (
-            _typedArray_: unknown,
+            _typedArray_: an Object that has a [[ViewedArrayBuffer]] internal slot,
```

Comment:
```suggestion
            _typedArray_: a TypedArray,
```
"has a [[ViewedArrayBuffer]] internal slot" is actually equivalent to "is a TypedArray or a DataView", but the algorithm assumes the existence of the `[[ArrayLength]]` slot, which rules out DataViews.